### ISSUE TITLE
--silent superseds --debug

### ISSUE THREAD

**Issue by budachst** [dadoonet/fscrawler#629]
Just a minor thing… when passing --silent and --debug to the cli, --silent superseds --debug. I'd rarther expect a warning to be issued, making the user make up his mind, especially since ctrl-c doesn't cause fscrawler to exit - it just keeps on running. ;)

**Comment by dadoonet** [comment]
> --silent superseds --debug

Yes. This is by design. But do you mean that `--debug` or `--trace` should overwrite `--silent` in all cases?

> especially since ctrl-c doesn't cause fscrawler to exit

Just tested with the latest SNAPSHOT and CTRL+C worked on my side. That said I did not launch a job and I know that there have been a bug in 2.5 which is normally fixed in 2.6-SNAPSHOT.

Related issues: #605 and #609
